wintermotes/index.html

```diff
@@ -20,5 +33,51 @@
 	<!-- 8. All messages must be printed in the order they where submitted -->
 
 	<div id="message-container"></div>
+
+	<script language="Javascript">
+	var messages = []
+	var container = document.getElementById('message-container')
+
+	function showMessage(){
+		var message = document.getElementById('messageValue')
+		
+		
+
+		if(validateString(message.value)){
+			container.innerHTML = container.innerHTML + "<p>" + message.value + "</p>"
+			messages.push(message.value)
+		} else {
+			console.log("String had number")
```

Comment:
This is properly because the acceptance criteria wasn't clear enough - sry about that.
We wanted to have it actually keep displaying a message regardless of it being an error.
These error-messages should also be rendered with a red color